docs/source/configuration.rst

```diff
@@ -1067,6 +1067,17 @@ If set to ``false``, this feature is disabled.
 
 If set to ``a resource name``, converse.js will synchronize only with a client that has that particular resource assigned to it.
 
+time_format
+-----------
+
+* Default: ``HH:MM``
+
+Valid option: ``HH:MM``, ``hh:mm``, ``hh:mm a``.
+
+This option makes the time format for the time shown, for each message, configurable. Converse can be configured to show 
+the time in 24hr or 12hr format. The default value gives a 24hr time-format. To change it to 12hr, use `hh:mm`, and to display
+AM/PM, use `hh:mm a`.
+
```

Comment:
Also the text here is too specific, it makes me think that you're still missing the point somewhat, I think because of the way the original ticket was specified.

What we are doing is letting the user specify **any** format for message time, they can even include the date if they want to. So don't say it's either 24hr or 12hr, just say that it can show any format that moment.js supports.